docs/source/examples/predict_turbofan_degredation.ipynb

```diff
@@ -369,6 +370,8 @@
    "outputs": [],
    "source": [
     "automl = evalml.AutoMLSearch(\n",
+    "    X_train,\n",
+    "    y_train,\n",
```

Comment:
For consistency, lets also use keyword arguments here.

Reply:
good catch